runtime/src/consensus/registry.rs

```diff
@@ -4,27 +4,324 @@
 //!
 //! This **MUST** be kept in sync with go/registry/api.
 //!
+use serde::{Deserialize, Serialize};
+use serde_repr::*;
+
 use crate::{
-    common::crypto::{hash, signature::SignatureBundle},
+    common::{
+        crypto::{
+            hash::Hash,
+            signature::{PublicKey, SignatureBundle},
+        },
+        namespace::Namespace,
+        quantity,
+        version::Version,
+    },
+    consensus::staking,
     storage::mkvs::WriteLog,
 };
-use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
```

Comment:
Move this to the top (before the serde imports) into a new group of imports (like we do elsewhere).

Reply:
Fixed :)  I thought that `make fmt` would sort the imports for me, but I guess we only have that on the Go side.